package.json

```diff
@@ -148,6 +148,11 @@
                         "Automatically restart the server",
                         "Do nothing"
                     ]
+                },
+                "clangd.detectAndDisableExtensionConflicts": {
```

Comment:
nit: how about `clangd.detectExtensionConflicts` + `Warn about conflicting extensions and suggest disabling them.`?